### ISSUE TITLE
i18n nextjs 14

### ISSUE THREAD

**Issue by pengxinseo** [cekocvetkov/software-crafting-chronicles#1]
Can i18n be added? Can velite be used with i18n to realize multi-language blog?

**Comment by cekocvetkov** [comment]
Absolutely, yes, one can use [next-intl](https://next-intl.dev/). To approach this. It even works with statically generated websites. I don't have it on my roadmap, but I have other projects, that I have successfully internationalized with the help of `next-intl`
